Pipfile

```diff
@@ -6,10 +6,10 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-ansible = "==2.9.9"
-molecule = "==3.0.4"
+ansible = "==2.9.21"
```

Comment:
I'd say we can start testing in newer versions, see the [recently updated nginx role for example](https://github.com/idealista/nginx_role/blob/master/Pipfile)

Reply:
Annotated for future releases / other roles